plugins/generic/externalFeed/templates/homepage.tpl

```diff
@@ -0,0 +1,50 @@
+{**
+* plugins/generic/externalFeed/templates/homepage.tpl
+*
+* Copyright (c) 2003-2013 John Willinsky
+* Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+*
+* Template for external feed content on journal homepage
+*}
+
+<div id="externalFeedsHome">
+
+	{foreach from=$entries item=entry}
+		<h3>{$entry.feedTitle}</h3>

```

Comment:
I'm not sure whether we want to dump these out unescaped -- will these be in spec if they include HTML? You could either pass through `|escape` (if it's not allowed to contain HTML) or `|strip_unsafe_html` (if we want to permit limited HTML).